pkg/content/file.go

```diff
@@ -35,15 +35,24 @@ type FileStore struct {
 	descriptor *sync.Map // map[digest.Digest]ocispec.Descriptor
 	pathMap    *sync.Map
 	tmpFiles   *sync.Map
+	ignoreNoName bool
```

Comment:
can you gofmt this file?

Reply:
That's funny. I thought I had. Clearly, I had not.

Updated. Ran `gofmt` on all of the files changed.